api/scripts/elastic_ops/pop_didx_boxytas/main.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+from elasticsearch import Elasticsearch
```

Comment:
Probably not needed for prod deploy, as we can elasticdump my local directory, or use the new document upload pipeline. But will leave in as a helpful script for now.